travel-sponsorship.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Contributor Travel Fund
+
+## Purpose
+
+To establish and administer a fund for members of the Kubernetes upstream contributor community to travel and particiate in KubeCon/CloudNativeCon.
+
+## Requirements
+
+ - Candidates must be a [member of the Kubernetes GitHub organization][membership] or working towards it. Exceptions: TODO
+ - Those requesting funds must indicate that they do not have funding available from another source, such as an employer or the event itself that might cover costs for presenters.
+
+## Process
+
+Submit an issue to a Kubernetes Travel Fund Issue-Only repository. 
+Your issue will go into a private repository where only TODO: $certain_group can see your information due to privacy.
+
+Requests must be made three months before the event. Requests submitted after that will not be approved.
+
+It is strongly recommended that you submit your request far enough in advance to get approval before making relevant expenditures.
+Review of requests will take at least 72 hours and may take considerably longer. Note, with limited funds available, approval is not guaranteed.
+
+### Visa Tips
+
+For all events, it is recommended to apply for a visa at least 6 months prior.
+
+### Requesting Travel Funds
+
+Open an issue template, the results go to a private repo that only TODO: $certain_group of people can see.
+        
+You'll be asked:
+ - Your Name
+ - The name of the event you plan to attend.
+ - The location, dates of the event and where you will be traveling from.
+ - The presentation you intend to give, if applicable.
+ - The size of the stipend you wish to receive.
+
+### Approval
+
+A request is approved when all of the following criteria are met:
+
+- The request has approval from at least two members of the TODO:$certain_group
+- No members of TODO: $certain_group objected to the request.
+- The issue has been closed.
+
+An appointed treasurer from the TODO: $certain_group will liaise with a member of CNCF on a regular basis to review the status of the travel fund. 
+In the event that any pending requests might not be approved because of available funds, a separate issue will be raised. 
+Generally speaking, however, members shouldn't need to worry about the status of the fund itself.
+
+### Considerations for approval of a request
+
+If there are too many requests for the limited funds, we will measure the following:
+
+#### Impact
+
+Preference is given to speakers, panelists, and those volunteering onsite at the conference.
+How much you contribute to the project.
```

Comment:
Do we ask them to write up how they contribute?

Reply:
Yeah and I think we ask someone who has attended previously to do one for us so we have an example of what a good one looks like as well.